src/apps/material-search/MaterialSearch.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC, useEffect } from "react";
+import { WorkId } from "../../core/utils/types/ids";
+import MaterialSearchListResults from "./MaterialSearchListResults";
+import MaterialSearchInputs from "./MaterialSearchInputs";
+import MaterialSearchPreview from "./MaterialSearchPreview";
+import useGetMaterialListSearch from "./useGetMaterialListSearch";
+import useGetSelectedMaterial from "./useGetSelectedMaterial";
+import useUpdateFields from "./useUpdateFields";
+
+type MaterialSearchProps = {
+  previouslySelectedWorkId: WorkId | null;
+  previouslySelectedMaterialType: string | null;
+  uniqueIdentifier: string;
+};
+
+const MaterialSearch: FC<MaterialSearchProps> = ({
+  previouslySelectedWorkId,
+  previouslySelectedMaterialType,
+  uniqueIdentifier
+}) => {
+  const {
+    availableMaterialTypes,
+    work,
+    isLoading: isSpecificMaterialLoading,
```

Comment:
I noticed that the hook is named `useGetSelectedMaterial`, but it returns work. Would it make sense to rename `work` to material for consistency, or perhaps rename the hook to useGetSelectedWork?

Also, I think renaming isLoading inside the hook could improve readability.

Reply:
Yeah good call 👍